.github/workflows/release.yml

```diff
@@ -0,0 +1,49 @@
+name: Changelog vs Release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Build Changelog
+        id: github_release
+        uses: mikepenz/release-changelog-builder-action@main
+        with:
+          configuration: ".github/changelog-configuration.json"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install JDK ${{ matrix.java_version }}
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'adopt'
+          java-version: 11
+      - name: Install Android SDK
+        uses: malinskiy/action-android/install-sdk@release/0.1.2
```

Comment:
Is this needed?
The Github `ubuntu-latest` machine already has the Android SDK installed.

e.g. the other workflow runs without it:
https://github.com/matomo-org/matomo-sdk-android/blob/7216a3227cd225abd2850ba9615f0d05194079ca/.github/workflows/pull-request-ci.yml#L14-L26

Reply:
Ok, I give it a try

Reply:
Indeed it's not needed

Reply:
Nice :slightly_smiling_face: